lib/util/miq-system.rb

```diff
@@ -396,8 +396,7 @@ def self.swapinfo
   #
   ##############################################################################################################################
   def self.disk_usage(file=nil)
-    file = nil if file.blank?
-    raise "file #{file} does not exist" unless File.exist?(file.to_s) || file.nil?
```

Comment:
Note, `File.exist?(file.to_s)`, specifically the `to_s` was probably added when File.exist? didn't accept Pathnames.  This shouldn't be needed anymore.